lib/html/pipeline/markdown_filter.rb

```diff
@@ -12,6 +12,8 @@ class Pipeline
     #
     # Context options:
     #   :gfm      => false    Disable GFM line-end processing
+    #   :commonmarker_extensions => [ ] Common marker extensions to
```

Comment:
What's the `[ ]` doing here? I think this is meant to suggest the default, which is instead written on the next line.